perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ResourcesManagerBlImpl.java

```diff
@@ -374,8 +381,17 @@ public void assignGroupToResource(PerunSession sess, Group group, Resource resou
 		assignGroupsToResource(sess, Collections.singletonList(group), resource);
 	}
 
+	@Override
+	public void assignGroupToResourceAsInactive(PerunSession perunSession, Group group, Resource resource) throws GroupResourceMismatchException, WrongReferenceAttributeValueException, WrongAttributeValueException {
+		assignGroupsToResource(perunSession, List.of(group), resource, GroupResourceStatus.INACTIVE);
+	}
+
 	@Override
 	public void assignGroupsToResource(PerunSession perunSession, Iterable<Group> groups, Resource resource) throws WrongAttributeValueException, WrongReferenceAttributeValueException, GroupResourceMismatchException {
+		assignGroupsToResource(perunSession, groups, resource, GroupResourceStatus.ACTIVE);
+	}
+
+	public void assignGroupsToResource(PerunSession perunSession, Iterable<Group> groups, Resource resource, GroupResourceStatus status) throws WrongAttributeValueException, WrongReferenceAttributeValueException, GroupResourceMismatchException {
```

Comment:
This method that takes the status should be available only at the impl layer. We don't want to allow anyone to change the status to 'FAILED' and 'ACTIVE' directly.

Reply:
For now set ACTIVE as default call to impl and removed the status attribute.